web/templates/recipient_row.hbs

```diff
@@ -27,11 +27,11 @@
             {{! topic link }}
             <a class="message_label_clickable narrows_by_topic tippy-narrow-tooltip"
               href="{{topic_url}}"
-              data-tippy-content="{{t 'Go to #{display_recipient} > {topic}' }}">
+              data-tippy-content="{{t 'Go to #{display_recipient} > {topic_display_name}' }}">
                 {{#if use_match_properties}}
                     <span class="stream-topic-inner">{{{match_topic}}}</span>
                 {{else}}
-                    <span class="stream-topic-inner">{{topic}}</span>
+                    <span class="stream-topic-inner {{#if is_empty_topic_name}}italic{{/if}}">{{topic_display_name}}</span>
```

Comment:
I think `empty-topic-display` would be a better class name to use than `italic`; that way, we can restyle topic display names freely in the future without editing the HTML.

Reply:
Makes sense. Made the changes.